api/v1beta3/cloudstackisolatednetwork_types.go

```diff
@@ -73,6 +73,9 @@ type CloudStackIsolatedNetworkStatus struct {
 	// Empty value means the network mode is NATTED, not ROUTED.
 	RoutingMode string `json:"routingMode,omitempty"`
 
+	// Firewall rules open status.
+	FirewallRulesOpened bool `json:"firewallRulesOpened,omitempty"`
```

Comment:
Is this field necessary to fix the issue or you are also trying to solve a visibility problem?